web_src/js/features/repo-branch.js

```diff
@@ -1,9 +1,18 @@
 import $ from 'jquery';
 
 export function initRepoBranchButton() {
-  $('.show-create-branch-modal.button').on('click', function () {
-    $('#create-branch-form')[0].action = $('#create-branch-form').data('base-action') + $(this).data('branch-from-urlcomponent');
-    $('#modal-create-branch-from-span').text($(this).data('branch-from'));
+  $('.show-create-branch-modal').on('click', function () {
+    let modalFormName = $(this).data('modal-form');
```

Comment:
https://docs.gitea.io/en-us/guidelines-frontend/#html-attributes-and-dataset

The guideline suggests to use `.attr()` instead of `.data()`

Reply:
Guidelines that were written after this code was written.